docs/contributing/first-time.md

```diff
@@ -163,20 +208,32 @@ Once you have completed, tested, and linted your code, and created a {ref}`contr
     ```shell
     git add <files>
     git commit -m "My commit message"
-    git push
+    git push origin my-branch-name
```

Comment:
```suggestion
    git push -u origin my-branch-name
```

This sets origin/my-branch-name as the upstream (remote) branch for this local branch, so that in the future you can run `git push` on this branch and it will know where to push it.

Reply:
@stevepiercy We still need this change adding `-u` so that the `git pull` below knows which branch to push to.

Reply:
Oops, went too fast. Suggestion committed.